RELEASING.md

```diff
@@ -14,20 +12,9 @@ Swiftly and the swiftly-install release script have different release schedules
 
 5. Create a tag on that commit with the format "x.y.z". Do not omit "z", even if its value is 0.
 
-6. Build the executables for the release by running ./scripts/build_release.sh from the root of the swiftly repository (do this once on an x86_64 machine and once on an aarch64 one)
+6. Build the executables for the release by running `swift run build-swiftly-release <version>` from the root of the swiftly repository
+  * Build on a Apple silicon macOS machine to produce a universal package for x86_64 and arm64
+  * Build on an Amazon Linux 2 image for x86_64
+  * Build on an Amazon Linux 2 image for arm64
```

Comment:
Is there a reason we are only building for Amazon Linux 2 image compared to all platforms? Also, should we use static sdk to build for multiple platforms?

Reply:
Amazon Linux 2 has the oldest glibc of the supported Linux distributions, so compiling with it will help guarantee binary compatibility with the rest. We have an issue open to use static Linux sdk / musl #119 for a future release. In the meantime time, the release needs to be made from AL2.